tests/kernel/kernel_info.cpp

```diff
@@ -44,15 +45,53 @@ TEST_CASE("Test kernel info", "[kernel]") {
 
   /** check program info parameters
    */
-  auto uint32Ret = kernel.get_info<sycl::info::kernel::num_args>();
-  check_return_type<uint32_t>(
-      uint32Ret, "sycl::kernel::get_info<sycl::info::kernel::num_args>()");
+  auto incorrect_num_args_usage = [&kernel] {
+    auto uint32Ret = kernel.get_info<sycl::info::kernel::num_args>();
+  };
+
+  {
+    INFO(
+        "Check that exception with error code \"errc::invalid\" is thrown in "
+        "case of invalid usage of "
+        "sycl::info::kernel::num_args descriptor");
+    CHECK_THROWS_MATCHES(incorrect_num_args_usage, sycl::exception,
+                         sycl_cts::util::equals_exception(sycl::errc::invalid));
+  }
 
-  auto stringRet = kernel.get_info<sycl::info::kernel::attributes>();
-  check_return_type<std::string>(
-      stringRet, "sycl::kernel::get_info<sycl::info::kernel::attributes>()");
+  auto incorrect_global_work_size_usage = [&kernel, &dev] {
+    auto range3Ret =
+        kernel.get_info<sycl::info::kernel_device_specific::global_work_size>(
+            dev);
+  };
+
+  {
+    INFO(
+        "Check exception with error code \"errc::invalid\" is thrown in case "
+        "of invalid usage of "
+        "sycl::info::kernel_device_specific::global_work_size descriptor");
```

Comment:
I guess the message should explain why it is invalid.

Reply:
Added.